### DISCUSSION TITLE
Show the branch names instead of the hash on branch compare

### DISCUSSION THREAD

**Discussion by euan-gwd** [gitkraken/vscode-gitlens#1360] [Q&A]
Hi
Thanks for all the hard work on this extension.
I just have on question, unable to find it in the docs.

How do I show the branch names instead of the hash on branch compare?

version 10.2.3

![Screenshot 2021-02-02 at 14 03 41](https://user-images.githubusercontent.com/13132625/106611541-0203c780-6560-11eb-9242-6bec11e5d6c5.png)

 version 11.2.0

![Screenshot 2021-02-02 at 14 02 55](https://user-images.githubusercontent.com/13132625/106611575-0af49900-6560-11eb-9fcd-dbc47846c5b9.png)

**Comment by eamodio** [comment]
I opened #1362 to track this issue.
